contents/handbook/company/team/paolo-damico.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: Paolo D'Amico's README
+title: Paolo D'Amico's Goals
 sidebar: Handbook
 showTitle: true
 ---
 
-## Goals (6 weeks until 6 Aug 2021)
```

Comment:
Q: Why remove the old context? :) Would be interesting to read a reflection as well.

Reply:
Makes sense, thanks Karl! Will share a reflection on previous goals in the future too (I have it in a separate doc, need to clean it up)